server/src/main/java/org/kaazing/gateway/server/GatewayCommandLineProcessor.java

```diff
@@ -97,6 +99,9 @@ private void launchGateway(String[] args, Properties properties) {
         final Gateway gateway = GatewayFactory.createGateway();
         gateway.setProperties(properties);
 
+        // Configure mina.netty/netty logging so that we can control logging using log4j-config.xml
+        InternalLoggerFactory.setDefaultFactory(new Slf4JLoggerFactory());
```

Comment:
May be just copy InternalLoggerFactory to mina.netty and change the default from JDK one to sl4j. That may be better since we don't have to change in Gateway.